src/Microsoft.AspNet.Mvc.ModelBinding/ModelStateDictionaryUtility.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Internal;
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    /// <summary>
+    /// Utility class for clearing entries in <see cref="ModelStateDictionary"/>.
+    /// </summary>
+    public static class ModelStateDictionaryUtility
+    {
+        /// <summary>
+        /// Clears <see cref="ModelStateDictionary"/>.
+        /// </summary>
+        /// <param name="modelState">The <see cref="ModelStateDictionary"/>.</param>
+        /// <param name="type">The <see cref="Type"/> of object whose properties entries have to be cleared from
+        /// <see cref="ModelStateDictionary"/>.</param>
+        public static void ClearModelStateDictionary([NotNull] ModelStateDictionary modelState, [NotNull] Type type)
+        {
+            var properties = type.GetProperties();
```

Comment:
It would seem that it would have to be passed in right? You can pass in a prefix to TryUpdateModel/TryValidateModel. If one isn't provided then it's the empty string